### PR TITLE
Allow optional single and double quotes in yaml versions/tags

### PR DESCRIPTION
**Pull request by mburumaxwell**
Fixes: #6313

### CODE REVIEW

File: docker/lib/dependabot/docker/file_updater.rb [hunk 1]
```diff
@@ -168,7 +168,7 @@ def update_helm(file)
         modified_content = file.content
 
         old_tags.each do |old_tag|
-          old_tag_regex = /^\s+(?:-\s)?(?:tag|version):\s+#{old_tag}(?=\s|$)/
+          old_tag_regex = /^\s+(?:-\s)?(?:tag|version):\s+["']{0,1}#{old_tag}["']{0,1}(?=\s|$)/
```
Comment (jeffwidman) [inline docker/lib/dependabot/docker/file_updater.rb:171]:
```suggestion
          old_tag_regex = /^\s+(?:-\s)?(?:tag|version):\s+["']?#{old_tag}["']?(?=\s|$)/
```

File: docker/spec/fixtures/helm/yaml/double-quotes.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+image:
+  repository: 'nginx'
+  tag: "1.14.2"
+  registry: registry.example.com
```
Comment (jeffwidman) [inline docker/spec/fixtures/helm/yaml/double-quotes.yaml:4]: Why not make all of these strings double-quoted? ie, `nginx` and `registry.example.com` could be double-quoted strings too...

File: docker/spec/fixtures/helm/yaml/single-quotes.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+image:
+  repository: 'nginx'
+  tag: '1.14.2'
+  registry: registry.example.com
```
Comment (jeffwidman) [inline docker/spec/fixtures/helm/yaml/single-quotes.yaml:4]: and single quote`registry.example.com` here?

**Review by jeffwidman — COMMENTED** [review]
Thanks for tackling this!

Spotted a couple of minor potential improvements.

File: docker/spec/fixtures/helm/yaml/double-quotes.yaml (hunk 2 above)
Comment (mburumaxwell) [reply docker/spec/fixtures/helm/yaml/double-quotes.yaml]: Done

File: docker/lib/dependabot/docker/file_updater.rb (hunk 1 above)
Comment (mburumaxwell) [reply docker/lib/dependabot/docker/file_updater.rb]: Done

File: docker/spec/fixtures/helm/yaml/single-quotes.yaml (hunk 3 above)
Comment (mburumaxwell) [reply docker/spec/fixtures/helm/yaml/single-quotes.yaml]: Done

**Comment by mburumaxwell** [comment]
> Thanks for tackling this!
> 
> Spotted a couple of minor potential improvements.

I have adjusted and rebased.

**Review by jeffwidman — APPROVED** [review]
👍

**Pull request merged**